app/domain/authentication/jwt/decoded_credentials.rb

```diff
@@ -6,21 +6,44 @@ class DecodedCredentials
       JWT_REQUEST_BODY_FIELD_NAME = "jwt".freeze
 
       def initialize(credentials)
+        @logger = Rails.logger
         @decoded_credentials = Hash[URI.decode_www_form(credentials)]
-        validate
+        validate_jwt_request_field_is_present
+        validate_body_contains_jwt
       end
 
       def jwt
-        @decoded_credentials[JWT_REQUEST_BODY_FIELD_NAME]
+        @jwt_token
```

Comment:
function name and variable should have the same name

Reply:
Chnaged